<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Aswinikumars-MacBook-Pro.local" name="Default test" tests="129" failures="29" timestamp="2 Mar 2022 16:28:27 GMT" time="4852.374" errors="0">
  <testcase name="beforeTest" time="471.744" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_Feed_Calls" time="1675.417" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_Homescreen_Adhesive_AdCall" time="0.76" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_NextGen_IM_AdCall" time="0.759" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_HourlyDetails_AdCall" time="0.784" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_TrendingPage_AdCall" time="0.753" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_Air_Quality_Details_AdCall" time="0.74" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_MapsDetails_AdCall" time="0.969" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_Health_Allergy_ContentPage_AdCall" time="0.778" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_Health_Running_ContentPage_AdCall" time="0.779" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_AlertCenterAd" time="0.954" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_SafetyAndPreparednessCard_AdCall" time="0.735" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_SafetyAndPreparednessCard_Call_Response" time="0.831" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="SafetyAndPreparedness call response doesn&amp;apos;t contains zMoatAdUnit=/7646/app_iphone_us/db_display/card/sp">
      <![CDATA[java.lang.AssertionError: SafetyAndPreparedness call response doesn't contains zMoatAdUnit=/7646/app_iphone_us/db_display/card/sp
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.Verify_Gampad_Call_ByResponseText(Utils.java:2285)
	at com.twc.ios.app.testcases.SmokeTest.Validate_SafetyAndPreparednessCard_Call_Response(SmokeTest.java:329)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_SafetyAndPreparednessCard_Call_Response -->
  <testcase name="validate_Marquee_sod_Custom_param" time="0.755" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_sod_Custom_param" time="0.759" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_zip_Custom_param" time="1.519" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_zip_Custom_param" time="1.622" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :zip value: 500045 is not matched with the expected value: 30303">
      <![CDATA[java.lang.AssertionError: Custom Parameter :zip value: 500045 is not matched with the expected value: 30303
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7817)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Feed1_zip_Custom_param(SmokeTest.java:412)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Feed1_zip_Custom_param -->
  <testcase name="Validate_Marquee_plat_Custom_param" time="0.811" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_plat_Custom_param" time="0.736" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_adid_Custom_param" time="0.726" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :adid value: nl is not matched with the expected value NotNull">
      <![CDATA[java.lang.AssertionError: Custom Parameter :adid value: nl is not matched with the expected value NotNull
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7938)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Marquee_adid_Custom_param(SmokeTest.java:477)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Marquee_adid_Custom_param -->
  <testcase name="Validate_Feed1_adid_Custom_param" time="0.811" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :adid value: nl is not matched with the expected value NotNull">
      <![CDATA[java.lang.AssertionError: Custom Parameter :adid value: nl is not matched with the expected value NotNull
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7938)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Feed1_adid_Custom_param(SmokeTest.java:490)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Feed1_adid_Custom_param -->
  <testcase name="Validate_Marquee_cnd_Custom_param" time="1.571" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_cnd_Custom_param" time="1.733" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :cnd value: clr is not matched with the expected value: sun">
      <![CDATA[java.lang.AssertionError: Custom Parameter :cnd value: clr is not matched with the expected value: sun
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7817)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Feed1_cnd_Custom_param(SmokeTest.java:529)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Feed1_cnd_Custom_param -->
  <testcase name="Validate_Marquee_dma_Custom_param" time="1.605" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_dma_Custom_param" time="1.576" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :dma value: nl is not matched with the expected value: 524">
      <![CDATA[java.lang.AssertionError: Custom Parameter :dma value: nl is not matched with the expected value: 524
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7817)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Feed1_dma_Custom_param(SmokeTest.java:568)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Feed1_dma_Custom_param -->
  <testcase name="Validate_Marquee_tmp_Custom_param" time="0.78" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_tmp_Custom_param" time="0.704" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_tmpc_Custom_param" time="1.399" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_tmpc_Custom_param" time="1.521" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :tmpc value: 25 is not matched with the expected value: 15">
      <![CDATA[java.lang.AssertionError: Custom Parameter :tmpc value: 25 is not matched with the expected value: 15
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7817)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Feed1_tmpc_Custom_param(SmokeTest.java:646)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Feed1_tmpc_Custom_param -->
  <testcase name="Validate_Marquee_tmpr_Custom_param" time="0.742" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_tmpr_Custom_param" time="0.685" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_ct_Custom_param" time="1.718" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_ct_Custom_param" time="2.072" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :ct value: Hyderabad is not matched with the expected value: Atlanta">
      <![CDATA[java.lang.AssertionError: Custom Parameter :ct value: Hyderabad is not matched with the expected value: Atlanta
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7817)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Feed1_ct_Custom_param(SmokeTest.java:724)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Feed1_ct_Custom_param -->
  <testcase name="Validate_Marquee_lang_Custom_param" time="0.843" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_lang_Custom_param" time="0.735" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_aid_Custom_param" time="0.806" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_aid_Custom_param" time="0.718" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_dynght_Custom_param" time="1.801" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_dynght_Custom_param" time="1.531" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :dynght value: N is not matched with the expected value: D">
      <![CDATA[java.lang.AssertionError: Custom Parameter :dynght value: N is not matched with the expected value: D
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7817)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Feed1_dynght_Custom_param(SmokeTest.java:844)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Feed1_dynght_Custom_param -->
  <testcase name="Validate_Marquee_locale_Custom_param" time="1.613" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_locale_Custom_param" time="1.498" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_tile_Custom_param" time="0.753" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_tile_Custom_param" time="0.704" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_ver_Custom_param" time="0.726" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_ver_Custom_param" time="0.732" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_ord_Custom_param" time="0.72" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_ord_Custom_param" time="0.714" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_rmid_Custom_param" time="0.699" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :rmid value: nl is not matched with the expected value NotNull">
      <![CDATA[java.lang.AssertionError: Custom Parameter :rmid value: nl is not matched with the expected value NotNull
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7938)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Marquee_rmid_Custom_param(SmokeTest.java:1026)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Marquee_rmid_Custom_param -->
  <testcase name="Validate_Feed1_rmid_Custom_param" time="0.701" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :rmid value: nl is not matched with the expected value NotNull">
      <![CDATA[java.lang.AssertionError: Custom Parameter :rmid value: nl is not matched with the expected value NotNull
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7938)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Feed1_rmid_Custom_param(SmokeTest.java:1039)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Feed1_rmid_Custom_param -->
  <testcase name="Validate_Marquee_st_Custom_param" time="1.445" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_st_Custom_param" time="1.637" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :st value: nl is not matched with the expected value: GA">
      <![CDATA[java.lang.AssertionError: Custom Parameter :st value: nl is not matched with the expected value: GA
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7817)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Feed1_st_Custom_param(SmokeTest.java:1078)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Feed1_st_Custom_param -->
  <testcase name="Validate_HomeScreen_Call_tf_Custom_param" time="0.785" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_DailyDetails_tf_Custom_param" time="0.727" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_daily_details_fcnd_Custom_param" time="1.687" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :fcnd value: clr is not matched with the expected value: sun">
      <![CDATA[java.lang.AssertionError: Custom Parameter :fcnd value: clr is not matched with the expected value: sun
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7817)
	at com.twc.ios.app.testcases.SmokeTest.Validate_daily_details_fcnd_Custom_param(SmokeTest.java:1143)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_daily_details_fcnd_Custom_param -->
  <testcase name="Validate_daily_details_fdynght_Custom_param" time="1.579" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :fdynght value: N is not matched with the expected value: D">
      <![CDATA[java.lang.AssertionError: Custom Parameter :fdynght value: N is not matched with the expected value: D
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7817)
	at com.twc.ios.app.testcases.SmokeTest.Validate_daily_details_fdynght_Custom_param(SmokeTest.java:1156)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_daily_details_fdynght_Custom_param -->
  <testcase name="Validate_daily_details_dt_Custom_param" time="0.749" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_daily_details_mnth_Custom_param" time="0.73" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_ltv_Custom_param" time="0.723" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_ltv_Custom_param" time="0.737" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_fhic_Custom_param" time="0.715" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_fhic_Custom_param" time="0.739" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_floc_Custom_param" time="0.731" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_floc_Custom_param" time="0.712" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_fltmpc_Custom_param" time="0.704" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_fltmpc_Custom_param" time="0.71" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_ftl_Custom_param" time="0.68" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_ftl_Custom_param" time="0.677" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_h2o_Custom_param" time="0.675" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_h2o_Custom_param" time="0.676" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_hmid_Custom_param" time="0.674" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_hmid_Custom_param" time="0.663" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_layer_Custom_param" time="0.668" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_layer_Custom_param" time="0.667" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_uv_Custom_param" time="0.664" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_uv_Custom_param" time="0.675" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_wind_Custom_param" time="0.668" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_wind_Custom_param" time="0.66" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="validate_Marquee_mr_Custom_param" time="0.656" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Feed1_mr_Custom_param" time="0.75" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Marquee_dnt_Custom_param" time="0.729" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :dnt value: y is not matched with the expected value n">
      <![CDATA[java.lang.AssertionError: Custom Parameter :dnt value: y is not matched with the expected value n
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7956)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Marquee_dnt_Custom_param(SmokeTest.java:1635)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Marquee_dnt_Custom_param -->
  <testcase name="Validate_Feed1_dnt_Custom_param" time="0.705" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :dnt value: y is not matched with the expected value n">
      <![CDATA[java.lang.AssertionError: Custom Parameter :dnt value: y is not matched with the expected value n
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7956)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Feed1_dnt_Custom_param(SmokeTest.java:1648)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Feed1_dnt_Custom_param -->
  <testcase name="Verify_AllergyDetails_Spotlight_AdCall" time="0.711" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="iu=%2F7646%2Fapp_iphone_us%2Fdb_display%2Fcontent%2Fallergy%2Ftips_sl ad call is not present">
      <![CDATA[java.lang.AssertionError: iu=%2F7646%2Fapp_iphone_us%2Fdb_display%2Fcontent%2Fallergy%2Ftips_sl ad call is not present
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.verifyPubadCal(Utils.java:437)
	at com.twc.ios.app.testcases.SmokeTest.Verify_AllergyDetails_Spotlight_AdCall(SmokeTest.java:1671)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Verify_AllergyDetails_Spotlight_AdCall -->
  <testcase name="Validate_AllergyDetails_Spotlight_pos_Custom_param" time="0.718" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Ad Call :iu=%2F7646%2Fapp_iphone_us%2Fdb_display%2Fcontent%2Fallergy%2Ftips_sl not found in charles session, hence Custom Parameter: pos validation skipped">
      <![CDATA[java.lang.AssertionError: Ad Call :iu=%2F7646%2Fapp_iphone_us%2Fdb_display%2Fcontent%2Fallergy%2Ftips_sl not found in charles session, hence Custom Parameter: pos validation skipped
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7919)
	at com.twc.ios.app.testcases.SmokeTest.Validate_AllergyDetails_Spotlight_pos_Custom_param(SmokeTest.java:1683)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_AllergyDetails_Spotlight_pos_Custom_param -->
  <testcase name="Validate_AllergyDetails_Spotlight_Ad_Size" time="0.701" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="iu=%2F7646%2Fapp_iphone_us%2Fdb_display%2Fcontent%2Fallergy%2Ftips_sl ad call is not present">
      <![CDATA[java.lang.AssertionError: iu=%2F7646%2Fapp_iphone_us%2Fdb_display%2Fcontent%2Fallergy%2Ftips_sl ad call is not present
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.verify_Ad_Size(Utils.java:1900)
	at com.twc.ios.app.testcases.SmokeTest.Validate_AllergyDetails_Spotlight_Ad_Size(SmokeTest.java:1693)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_AllergyDetails_Spotlight_Ad_Size -->
  <testcase name="Verify_Daily_Details_AdUnits" time="297.945" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_Map_Details_mr_Custom_Parameter" time="113.504" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_enable_preConditions_for_WeatherFx_API_Parameters" time="591.648" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_MapDetails_wfxtg_Custom_param" time="1.1" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_MapDetails_cxtg_Custom_param" time="0.846" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_MapDetails_zcs_Custom_param" time="0.89" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_MapDetails_hzcs_Custom_param" time="0.941" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_MapDetails_nzcs_Custom_param" time="0.829" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_enable_preConditions_for_WeatherFx_API_ParametersFromPlanningCard" time="327.658" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_DailyDetails_Banner_Ad_wfxtg_Custom_param" time="0.931" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_DailyDetails_Banner_Ad_cxtg_Custom_param" time="0.807" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_DailyDetails_Banner_Ad_zcs_Custom_param" time="0.762" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_DailyDetails_Banner_Ad_hzcs_Custom_param" time="0.762" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_DailyDetails_Banner_Ad_nzcs_Custom_param" time="0.797" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_PrerollAd" time="200.612" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_PreRollVideo_cmsid_Custom_param" time="0.304" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_PreRollVideo_ttid_Custom_param" time="0.156" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_PreRollVideo_vid_Custom_param" time="0.177" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_PreRollVideo_description_url_Custom_param" time="0.15" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_PreRollVideo_ref_Custom_param" time="0.159" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_PreRollVideo_dnt_Custom_param" time="0.146" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="Custom Parameter :dnt value: y is not matched with the expected value n">
      <![CDATA[java.lang.AssertionError: Custom Parameter :dnt value: y is not matched with the expected value n
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_custom_param_val_of_gampad(Utils.java:7956)
	at com.twc.ios.app.testcases.SmokeTest.Validate_PreRollVideo_dnt_Custom_param(SmokeTest.java:2486)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_PreRollVideo_dnt_Custom_param -->
  <testcase name="Validate_PreRollVideo_wfxtg_Custom_param" time="0.154" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_PreRollVideo_cxtg_Custom_param" time="0.144" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_PreRollVideo_zcs_Custom_param" time="0.153" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_PreRollVideo_hzcs_Custom_param" time="0.147" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_PreRollVideo_nzcs_Custom_param" time="0.139" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_IMA_SDK_version" time="0.142" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_Hourly_Details_BigAd1_Adcall" time="166.278" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_Hourly_Details_BigAd2_Adcall" time="0.188" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_Hourly_Details_BigAd3_Adcall" time="0.067" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="kill_and_Launch_app_for_api_calls" time="111.139" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_Lotame_Call_On_KillAndLaunch" time="0.169" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="bcp.crwdcntrl.net ad call is not present">
      <![CDATA[java.lang.AssertionError: bcp.crwdcntrl.net ad call is not present
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.verifyAPICal(Utils.java:1318)
	at com.twc.ios.app.testcases.SmokeTest.Verify_Lotame_Call_On_KillAndLaunch(SmokeTest.java:2649)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Verify_Lotame_Call_On_KillAndLaunch -->
  <testcase name="Verify_TurboAPI_Call_On_KillAndLaunch" time="0.166" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_WFXTriggers_Call_On_KillAndLaunch" time="0.156" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Verify_LocationWFXTriggers_Call_On_KillAndLaunch" time="0.158" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Amazon_SDK_adsdk_parameter" time="0.151" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="amazon-adsystem.com/e/msdk/ads? call is not present in Charles session, hence Custom Parameter: adsdk validation skipped">
      <![CDATA[java.lang.AssertionError: amazon-adsystem.com/e/msdk/ads? call is not present in Charles session, hence Custom Parameter: adsdk validation skipped
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_Amazon_aax_call_parameter(Utils.java:10961)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Amazon_SDK_adsdk_parameter(SmokeTest.java:2702)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Amazon_SDK_adsdk_parameter -->
  <testcase name="Validate_GMA_SDK_version" time="0.157" classname="com.twc.ios.app.testcases.SmokeTest"/>
  <testcase name="Validate_Criteo_SDK_config_app_Call_sdkVersion_parameter" time="0.152" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="bidder.criteo.com/config/app call is not present in Charles session, hence Custom Parameter: sdkVersion validation skipped">
      <![CDATA[java.lang.AssertionError: bidder.criteo.com/config/app call is not present in Charles session, hence Custom Parameter: sdkVersion validation skipped
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_Criteo_SDK_config_app_call_parameter(Utils.java:9082)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Criteo_SDK_config_app_Call_sdkVersion_parameter(SmokeTest.java:2729)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Criteo_SDK_config_app_Call_sdkVersion_parameter -->
  <testcase name="Validate_Criteo_SDK_Bidder_API_Call_Response_Code" time="0.154" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="bidder.criteo.com/inapp/v2 call is not present in Charles session, hence response code validation is failed">
      <![CDATA[java.lang.AssertionError: bidder.criteo.com/inapp/v2 call is not present in Charles session, hence response code validation is failed
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.verifyCriteo_inapp_v2_Call_ReponseStatusCode(Utils.java:11251)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Criteo_SDK_Bidder_API_Call_Response_Code(SmokeTest.java:2742)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Criteo_SDK_Bidder_API_Call_Response_Code -->
  <testcase name="Validate_Criteo_SDK_Initialization_API_Call_Response_Code" time="0.164" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="bidder.criteo.com/config/app call is not present in Charles session, hence response code validation is failed">
      <![CDATA[java.lang.AssertionError: bidder.criteo.com/config/app call is not present in Charles session, hence response code validation is failed
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.verifyCriteo_config_app_Call_ReponseStatusCode(Utils.java:11295)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Criteo_SDK_Initialization_API_Call_Response_Code(SmokeTest.java:2755)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Criteo_SDK_Initialization_API_Call_Response_Code -->
  <testcase name="Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_csmEnabled" time="0.175" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="bidder.criteo.com/config/app call is not present in Charles session, hence Custom Parameter: csmEnabled validation skipped">
      <![CDATA[java.lang.AssertionError: bidder.criteo.com/config/app call is not present in Charles session, hence Custom Parameter: csmEnabled validation skipped
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_Criteo_SDK_config_app_call_response_parameter(Utils.java:11549)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_csmEnabled(SmokeTest.java:2770)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_csmEnabled -->
  <testcase name="Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_liveBiddingEnabled" time="0.151" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="bidder.criteo.com/config/app call is not present in Charles session, hence Custom Parameter: liveBiddingEnabled validation skipped">
      <![CDATA[java.lang.AssertionError: bidder.criteo.com/config/app call is not present in Charles session, hence Custom Parameter: liveBiddingEnabled validation skipped
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_Criteo_SDK_config_app_call_response_parameter(Utils.java:11549)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_liveBiddingEnabled(SmokeTest.java:2785)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_liveBiddingEnabled -->
  <testcase name="Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_prefetchOnInitEnabled" time="0.15" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="bidder.criteo.com/config/app call is not present in Charles session, hence Custom Parameter: prefetchOnInitEnabled validation skipped">
      <![CDATA[java.lang.AssertionError: bidder.criteo.com/config/app call is not present in Charles session, hence Custom Parameter: prefetchOnInitEnabled validation skipped
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_Criteo_SDK_config_app_call_response_parameter(Utils.java:11549)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_prefetchOnInitEnabled(SmokeTest.java:2820)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_prefetchOnInitEnabled -->
  <testcase name="Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_remoteLogLevel" time="0.148" classname="com.twc.ios.app.testcases.SmokeTest">
    <failure type="java.lang.AssertionError" message="bidder.criteo.com/config/app call is not present in Charles session, hence Custom Parameter: remoteLogLevel validation skipped">
      <![CDATA[java.lang.AssertionError: bidder.criteo.com/config/app call is not present in Charles session, hence Custom Parameter: remoteLogLevel validation skipped
	at org.testng.Assert.fail(Assert.java:94)
	at com.twc.ios.app.general.Utils.validate_Criteo_SDK_config_app_call_response_parameter(Utils.java:11549)
	at com.twc.ios.app.testcases.SmokeTest.Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_remoteLogLevel(SmokeTest.java:2835)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validate_Criteo_SDK_Initialization_API_Call_Response_Parameter_remoteLogLevel -->
  <testcase name="Verify_Seasonal_Hub_Details_Page_AdCall" time="755.807" classname="com.twc.ios.app.testcases.SmokeTest"/>
</testsuite> <!-- Default test -->
